projects/kalidea/kaligraphi/src/lib/02-form/kal-select/kal-select.component.ts

```diff
@@ -296,8 +297,12 @@ export class KalSelectComponent
    */
   writeValue(value: any) {
     Promise.resolve().then(() => {
-      this.select(value);
-      super.writeValue(value);
+      if (!this.hasDefaultValue) {
+        this.select(value);
+        super.writeValue(value);
+
+        this.hasDefaultValue = false;
+      }
```

Comment:
If `!this.hasDefaultValue` is true 
then `this.hasDefaultValue = false;` is redundant